client/layout/utils.ts

```diff
@@ -49,10 +49,14 @@ export const handleScroll = ( event: React.UIEvent< HTMLElement > ): void => {
 
 		if (
 			windowHeight >= secondaryElHeight + masterbarHeight &&
-			content.style.minHeight !== 'initial'
+			secondaryEl !== undefined &&
+			secondaryEl !== null &&
+			( content.style.minHeight !== 'initial' || secondaryEl.style.position )
```

Comment:
We seem to be doing more thorough checks on `secondaryEl` than we are on `content` when they are being operated on in similar ways

Do you think we should apply the same check as the above comment (`content instanceof HTMLElement`) to `content`?

Reply:
Fixed in 556bd9b72e9bd7282114cf19422d2f6a47100883